testapi.pm

```diff
@@ -1477,6 +1483,42 @@ sub mouse_set ($mx, $my) {
     query_isotovideo('backend_mouse_set', {x => $mx, y => $my});
 }
 
+=head2 mouse_move
+
+  mouse_move($x, $y, [$pixels]);
+
+Slowly move mouse pointer to given coordinates
+
+=cut
+
+sub mouse_move ($x, $y, $pixels = 5) {
+    my $old = query_isotovideo('backend_get_last_mouse_set');
+    my $curx = $old->{x};
+    my $cury = $old->{y};
+    return if $curx < 0 or $cury < 0;
+    while (1) {
```

Comment:
Generally it would be nice to have this whole loop within the backend (to avoid the IPC/serialization overhead for every animation step) but I guess for simplicity we can optimize that later.

Reply:
Yeah, I mentioned this in the PR description.